lib/spack/spack/build_systems/cmake.py

```diff
@@ -32,6 +32,50 @@
 from spack.package import PackageBase, run_after
 
 
+def processCMakeCacheFile(cache_file):
+    def determineLineInterest(line):
+        return not line.startswith("//") \
+            and not line.startswith("#") \
+            and len(line) > 0 \
+            and not ":INTERNAL" in line \
+            and not ":PATH" in line \
+            and not ":FILEPATH" in line \
+            and not ":STATIC" in line \
+            and not "CMAKE_" in line \
+            and line.count("/") < 4  # *really* don't want to capture paths
```

Comment:
I really want to avoid creating a package which does something like

variant(CMAKE_INSTALL_PREFIX="/my/silly/home/directory"). The types are an attempt at this, but these could be encoded as type STRING....

I'm not really attached to this line, would like to know whether it's worth having